soroban-env-host/src/test/invocation.rs

```diff
@@ -29,6 +29,34 @@ fn invoke_single_contract_function() -> Result<(), HostError> {
     Ok(())
 }
 
+#[test]
+fn invoke_alloc() -> Result<(), HostError> {
```

Comment:
Now that we have the alloc wasm, it would be nice to also add a test that tries to allocate a lot of memory via `alloc` (probably belongs to the 'hostile' test set).

Reply:
We [already have a test](https://github.com/stellar/rs-soroban-env/blob/main/soroban-env-host/src/test/hostile.rs#L195-L247) in `hostile.rs` that directly calls the wasm memory.grow opcode that alloc calls.